src/code/z_camera.c

```diff
@@ -159,29 +263,29 @@ s16 Camera_AngleDiffAndScale(s16 angle1, s16 angle2, f32 scale) {
 }
 
 /**
- * Calculates the current offset between the camera's at-coordinates and the centered actor's coordinates
+ * Calculates the current offset between the camera's at-coordinates and the focal actors coordinates
  */
-void Camera_UpdateAtActorOffset(Camera* camera, Vec3f* actorOffset) {
-    camera->atActorOffset.x = camera->at.x - actorOffset->x;
-    camera->atActorOffset.y = camera->at.y - actorOffset->y;
-    camera->atActorOffset.z = camera->at.z - actorOffset->z;
+void Camera_SetFocalActorAtOffset(Camera* camera, Vec3f* focalActorPos) {
```

Comment:
Not sure what the solution is, but using the variable `at` in the name like this is confusing if you aren't familiar with the naming. Maybe just drop `at` from this name?

Reply:
Similar to before, there are different types of focal actor offsets variables, so `FocalActorOffset` would be far to generic. I think anyone who wants to work with camera needs to quickly pick up the terms `eye` and `at`